src/less.plugin.coffee

```diff
@@ -38,21 +38,10 @@ module.exports = (BasePlugin) ->
 				file.setMetaDefaults('referencesOthers': true)  if opts.content.indexOf('@import') isnt -1
 
 				# Parse
-				new (less.Parser)(parseOptions).parse opts.content, (err,tree) ->
-					# Check
+				less.render opts.content, parseOptions, (err, output) ->
 					if err
-						err = new Error(less.formatError(err, parseOptions))
 						return next(err)
```

Comment:
I believe this makes the error a lot more readable, such as including the line numbers of input source and sometimes a code sample